### ISSUE TITLE
Section508.gov - Reporting Accessibility with VPAT 2 - A Guide for Vendors

### ISSUE THREAD

**Issue by mgifford** [GSA/openacr#313]
3rd most popular search for VPAT https://www.section508.gov/event/reporting-accessibility-vpat-2-guide-vendors/

This just needs to be archived.

That said, there are going to need to be a series of events or recordings to introduce OpenACR

**Comment by danid123** [comment]
@mgifford  It looks like they still have active events on the events page since they have something for March 2022, so maybe the direction is to add OpenACR events to the events calendar?

**Comment by mgifford** [comment]
Yes, but I do think some of the older events might also be removed in order to just avoid confusion. Do we want to have to edit the content of an old event page?

**Comment by mgifford** [comment]
See https://github.com/GSA/Section508.gov/pull/229
